io/net/virt-net/network_virtualization.py

```diff
@@ -78,8 +78,11 @@ def setUp(self):
         cmd = 'lssyscfg -r sys  -F name'
         output = self.session_hmc.cmd(cmd)
         self.server = ''
+        lpar_split = self.lpar.split('-', maxsplit=1)
+        if(len(lpar_split) != 2):
+            self.cancel("LPAR name incorrectly formatted. Expecting: <name>-lp<#>")
```

Comment:
@cforno12 I don't think it is right to force lpar names. <name>-lp<#> is just a common nomenclature we follow.